chaosoci/compute/common.py

```diff
@@ -0,0 +1,53 @@
+from typing import Any, List, Dict
+
+from oci.core import ComputeClient
+
+from chaoslib.exceptions import ActivityFailed
+
+__all__ = ["get_instances", "filter_instances"]
+
+
+def get_instances(client: ComputeClient = None,
+                  compartment_id: str = None) -> List:
+    """Return a complete, unfiltered list of instances in the compartment."""
+    instances = []
+
+    instances_raw = client.list_instances(compartment_id=compartment_id)
+    instances.extend(instances_raw.data)
+    while instances_raw.has_next_page:
+        instances_raw = client.list_instances(compartment_id=compartment_id,
+                                              page=instances_raw.next_page)
+        instances.extend(instances_raw.data)
+
+    return instances
+
+
+def filter_instances(instances: List = None,
+                     filters: Dict[str, Any] = None) -> List:
```

Comment:
same question :)

Reply:
Thought about it better; it is better to know precisely what we are returning; fixed.